src/app/auth/login/page.tsx

```diff
@@ -0,0 +1,88 @@
+'use client';
+
+import { useState } from 'react';
+import { useRouter } from 'next/navigation';
+import TextInput from '@/components/TextInput/index';
+import { H1, H4Centered, P, AColored } from '@/styles/text';
+import supabase from '@/api/supabase/createClient';
+import COLORS from '@/styles/colors';
+import { FormDiv, QuestionsDiv, SixteenDiv } from '../styles';
+
+export default function Login() {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  // const [errorMessage, setErrorMessage] = useState('');
+  const { push } = useRouter();
+  // commenting out Sign Up to use in the next PR -> create a separate sign in page
+  /*
+  const handleSignUp = async () => {
+    const { error } = await supabase.auth.signUp({
+      email,
+      password,
+    });
+
+    if (error) {
+      throw new Error(`An error occurred trying to sign up: ${error.message}`);
+    }
+    push('/');
+  };
+  */
+
+  const handleSignIn = async () => {
+    const { error } = await supabase.auth.signInWithPassword({
+      email,
+      password,
+    });
+
+    if (error) {
+      throw new Error(`An error occurred trying to sign in: ${error.message}`);
+    }
+    push('/');
+  };
+
+  return (
+    <FormDiv>
+      <H1>Log In</H1>
+      <SixteenDiv>
+        <QuestionsDiv>
+          <TextInput
+            label="Email"
+            placeholder="email@example.com"
+            erroring={false}
+            errorText="Email Error" // {errorMessage}
+            type="email"
+            name="email"
+            value={email}
+            setValue={setEmail}
+          />
+          <TextInput
+            label="Password"
+            placeholder="Password"
+            erroring={false}
+            errorText="Password Error"
+            type="password"
+            name="password"
+            value={password}
+            setValue={setPassword}
+          />
+        </QuestionsDiv>
+        <P>
+          <AColored $color={COLORS.midDarkGrey} href="/reset-password">
+            Forgot your password?
+          </AColored>
+        </P>
+      </SixteenDiv>
```

Comment:
nit: prob rename from sixteen to like spacer div just so it's a lil bit easier to understand when you first see it

Reply:
gotcha! i've renamed it!